src/App.js

```diff
@@ -1,10 +1,23 @@
-import React from 'react';
+import React from "react";
+import VoteBox from "./components/VoteBox";
+import styled from "styled-components";
```

Comment:
이렇게 분리해서 정리해주시면 보기 좋을 것 같아요🤗

```suggestion
import React from "react";
import styled from "styled-components";

import VoteBox from "./components/VoteBox";
```